events/wc_titanforged_events.txt

```diff
@@ -1 +1,218 @@
 namespace = WCUTF
+character_event = {
+	#Basic event information
+	id = WCUTF.1
+	desc = {
+		trigger = {
+			character = 400501
+		}
+		text = EVTDESC_WCUTF.1_loken
+	}
+
+	desc = {
+		trigger = {
+			character = 400502
+		}
+		text = EVTDESC_WCUTF.1_thorim
+	}
+	picture = GFX_evt_thorim_sif
+	border = GFX_event_normal_frame_intrigue
+	
+	#Fast event triggers
+	only_playable = yes
+	is_triggered_only = yes
+	ai = no
+	
+
+	trigger = {
+		OR = {
+			character = 400501
+			character = 400502
+		}
+		is_save_game = no
+	}
+
+	immediate = {
+		c_400502 = {
+			save_event_target_as = target_thorim
+		}
+	}
+	option = {
+		name = EVTOPTA_WCUTF.1
+	}
```

Comment:
The option is very... tricky. Not blocked for the players though it makes the characters they started unplayably.